pkgs/applications/window-managers/sway/osd.nix

```diff
@@ -0,0 +1,38 @@
+{ lib
+, rustPlatform
+, fetchFromGitHub
+, pkg-config
+, gtk3
+, gtk-layer-shell
+, libpulseaudio
+}:
+
+rustPlatform.buildRustPackage {
+  pname = "swayosd";
+  version = "unstable-2023-05-09";
+
+  src = fetchFromGitHub {
+    owner = "ErikReider";
+    repo = "SwayOSD";
+    rev = "5c2176ae6a01a18fdc2b0f5d5f593737b5765914";
+    hash = "sha256-rh42J6LWgNPOWYLaIwocU1JtQnA5P1jocN3ywVOfYoc=";
+  };
+
+  cargoHash = "sha256-/va3ES6HqFLh3+YFjyj8/x65wvkW0cU9v3fl6pEQ084=";
```

Comment:
```bash
error: hash mismatch in fixed-output derivation '/nix/store/dgr665lippyx0w5wbi2arlngl7gzn3wz-swayosd-unstable-2023-05-09-vendor.tar.gz.drv':
         specified: sha256-/va3ES6HqFLh3+YFjyj8/x65wvkW0cU9v3fl6pEQ084=
            got:    sha256-ZcgrUcRQTcEYhw2mpJDuYDz3I/u/2Q+O60ajXYRMeow=
error: 1 dependencies of derivation '/nix/store/468kg6b16bdhy9563jy8czcziiaz8qb4-swayosd-unstable-2023-05-09.drv' failed to build
```

Reply:
You are right, I got the same hash this time. No idea what happened but I'll edit it for now.